library/src/main/java/com/schibsted/spain/barista/BaristaAssertions.java

```diff
@@ -162,6 +163,10 @@ public static void assertHint(@IdRes int id, String text) {
     onView(withId(id)).check(matches(withHint(text)));
   }
 
+  public static void assertDrawable(@IdRes int id) {
+    onView(withId(id)).check(matches(withDrawable(DrawableMatcher.Companion.getANY())));
```

Comment:
That `getANY()` hurts my lovely new eyes 👀 but I don't know how to do it better

Reply:
Hi @rocboronat !
You can annotate `DrawableMatcher` as this:

```kotlin
companion object {
    private val EMPTY = -1
    @JvmStatic
    @get:JvmName("any")
    val ANY = -2
  }
```
And then you can use it from java like this:
```kotlin
DrawableMatcher.any()
```

Reply:
Woooow thanks for sharing! Now, I don't know if I like more the problem or the solution 😄 Let's wait for more feedback... BTW @SmasSive we need you at #159 !

Reply:
@Sloy we need your feedback. I personally think that both solutions are not too pretty 😅

Reply:
I think it's the price you have to pay to live with Java. You can still use only the annotation `@JvmStatic` to avoid the use of `Companion` but you will have to use `getANY()`.
Let's see what @Sloy has to say about it...

Reply:
I was trying to understand this PR yesterday afternoon, but wasn't a good moment for that :D
Please give me some time to finish some other things and I will be back 🤖

Reply:
Take your time 😏😘

Reply:
@Sloy not too much time, please... :·/

Reply:
Hi! Missed me? 💋  
Sorry 🙏 

**Here's my opinion:**
EMPTY and ANY look like internal constants that we should avoid exposing to the outside world. If we wanted to make them public we should use an enum or something more helpful.

**Here's my proposal:**
- Keep the constants privates to avoid leaking them to the outside.
- Make the DrawableMatcher constructor private.
- Provide public static factory methods in the companion object for the three kinds of matchers: `withId(drawableId)`, `any()`, `empty()`.

Reply:
Hi! Missed me? 💋  
Sorry 🙏 

**Here's my opinion:**
EMPTY and ANY look like internal constants that we should avoid exposing to the outside world. If we wanted to make them public we should use an enum or something more helpful.

**Here's my proposal:**
- Keep the constants privates to avoid leaking them to the outside.
- Make the DrawableMatcher constructor private.
- Provide public static factory methods in the companion object for the three kinds of matchers: `withId(drawableId)`, `any()`, `empty()`.